dashboard/app/controllers/programming_environments_controller.rb

```diff
@@ -17,6 +17,7 @@ def update
     programming_environment.assign_attributes(programming_environment_params)
     begin
       programming_environment.save! if programming_environment.changed?
+      programming_environment.write_serialization
```

Comment:
Should you only write the serialization if something changed?

Reply:
oh good thought!